package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "pug-addon",
   "description": "Augmented Reality Game Experience",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
You changed the package.json `version`, but do we also need to modify the addon/webextension/manifest.json `version` string?

https://github.com/gregglind/addon-wr/blob/191d7b000265e174d38a5bcc77d76e67a792dd9e/addon/webextension/manifest.json#L3